manifests/AgileBits/1Password/7.4.767.yaml

```diff
@@ -0,0 +1,15 @@
+Id: AgileBits.1Password
+Version: 7.4.767
+Name: 1Password
+Publisher: AgileBits
+License: AgileBis, Inc. Terms of Service
+LicenseUrl: https://1password.com/legal/terms-of-service/
+AppMoniker: 1password
+Tags: password
+Description: 1Password remembers them all for you. Save your passwords and log in to sites with a single click. It's that simple.
```

Comment:
Hey @alanlima , echoing what @galich said, thank you for putting this up. Could you please update this description to the following:

1Password remembers all your passwords for you, and keeps them safe and secure behind the one password that only you know.